src/ivory-px4-hw/calibration.conf

```diff
@@ -1,8 +1,32 @@
-[calibration.accelerometer]
-x_offset=0
-y_offset=0
-z_offset=0
-[calibration.magnetometer]
-x_offset=0
-y_offset=0
-z_offset=0
+[calibration.mpu6000.accelerometer]
+x_offset=0.0
+y_offset=0.0
+z_offset=0.0
+x_scale=1.0
+y_scale=1.0
+z_scale=1.0
+[calibration.mpu6000.gyroscope]
+x_offset=0.0
+y_offset=0.0
+z_offset=0.0
+x_scale=1.0
+y_scale=1.0
+z_scale=1.0
+[calibration.lsm303d.accelerometer]
```

Comment:
It's easier to keep it in, because other configurations use it. We would have to special-case the code otherwise.